starter-code/app.js

```diff
@@ -6,48 +10,48 @@ const cookieParser   = require('cookie-parser');
 const bodyParser     = require('body-parser');
 const expressLayouts = require('express-ejs-layouts');
 const mongoose       = require('mongoose');
-const index          = require('./routes/index');
-const users          = require('./routes/users');
-const drones         = require('./routes/drones');
 
-mongoose.connect('mongodb://localhost/drones-dev');
+require('./config/db.config'); // Import DB config - 
```

Comment:
sí es correcto, justo acabáis de darlo en clase, es una forma de embeber el archivo a esta algura, lo puedes ver de esta forma, todo lo que tenga el otro archivo, irá aquí en este punto